tests/test_credentials.py

```diff
@@ -0,0 +1,17 @@
+from boto3.session import Session
+from moto import mock_s3
+
+from prefect_aws.credentials import AwsCredentials
+
+
+def test_aws_credentials_get_boto3_session(aws_credentials):
```

Comment:
Since we're creating a fresh instance of aws_credentials (rather than using the one in conftest)
```suggestion
def test_aws_credentials_get_boto3_session():
```